### DISCUSSION TITLE
[v3][Forms] Checkboxlist documentation errors

### DISCUSSION THREAD

**Discussion by yagogak** [filamentphp/filament#8903] [General]
The CheckboxList documentation https://filamentphp.com/docs/3.x/forms/fields/checkbox-list got some errors on relationship methods.

```
CheckboxList::make('technologies')
    ->relationship(titleAttribute: 'name')
```
the correct syntax is :

```
CheckboxList::make('places')
                        ->relationship('technologies', 'name')
```
so the part for Customizing the relationship query is also wrong.

**Comment by leandrocfe** [comment]
[send a PR](https://github.com/filamentphp/filament/edit/3.x/packages/forms/docs/03-fields/06-checkbox-list.md) please
